src/pages/Add.js

```diff
@@ -9,59 +9,70 @@ function Add() {
       question: question,
       answerOne: answerOne,
       answerTwo: answerTwo,
       answerThree: answerThree
     };
-    alert(JSON.stringify(poll));
+    const response = await fetch("http://localhost:4000/polls", {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json"
+      },
+      body: JSON.stringify(poll)
+    });
+    const createdPoll = await response.json();
+    alert(`Created poll with the id ${createdPoll.id}`);
   }
   return (
-    <Card>
-      <form className="add-form" onSubmit={handleSubmit}>
-        <input
-          className="add-form__input-question"
-          type="text"
-          placeholder="Enter your question..."
-          value={question}
-          onChange={event => {
-            setQuestion(event.target.value);
-          }}
-        />
-        <input
-          className="add-form__input"
-          type="text"
-          placeholder="First answer"
-          value={answerOne}
-          onChange={event => {
-            setAnswerOne(event.target.value);
-          }}
-        />
-        <input
-          className="add-form__input"
-          type="text"
-          placeholder="Second answer"
-          value={answerTwo}
-          onChange={event => {
-            setAnswerTwo(event.target.value);
-          }}
-        />
-        <input
-          className="add-form__input"
-          type="text"
-          placeholder="Third answer"
-          value={answerThree}
-          onChange={event => {
-            setAnswerThree(event.target.value);
-          }}
-        />
-        {/* Add <Link to="/vote">Vote!!</Link> */}
-        <Button>Create Poll</Button>
-      </form>
-    </Card>
+    <>
+      <Card>
+        <form className="add-form" onSubmit={handleSubmit}>
+          <input
+            className="add-form__input-question"
+            type="text"
+            placeholder="Enter your question..."
+            value={question}
+            onChange={event => {
+              setQuestion(event.target.value);
+            }}
+          />
+          <input
+            className="add-form__input"
+            type="text"
+            placeholder="First answer"
+            value={answerOne}
+            onChange={event => {
+              setAnswerOne(event.target.value);
+            }}
+          />
+          <input
+            className="add-form__input"
+            type="text"
+            placeholder="Second answer"
+            value={answerTwo}
+            onChange={event => {
+              setAnswerTwo(event.target.value);
+            }}
+          />
+          <input
+            className="add-form__input"
+            type="text"
+            placeholder="Third answer"
+            value={answerThree}
+            onChange={event => {
+              setAnswerThree(event.target.value);
+            }}
+          />
+          {/* Add <Link to="/vote">Vote!!</Link> */}
+        </form>
+      </Card>
+
+      <Button>Create Poll</Button>
```

Comment:
the Create Poll Button has to be inside the form to be working. only then it is a submit button. How else do you want to submit the new poll then?